build-system/tasks/dist.js

```diff
@@ -153,6 +153,7 @@ async function dist() {
     console.log('\n');
   }
 
+  await generateFileListing();
```

Comment:
Move the call to `generateFileListing()` to line 167 so that the files (optionally) generated by lines 162-164 are also included in the listing.